### PR TITLE
Add error note

### PR DESCRIPTION
**Pull request by martinhaiss**
This resolves [#11](https://github.com/martinhaiss/capstone-project/issues/25)

An error note is displayed, when the joined-category is empty.

### CODE REVIEW

File: src/components/Error.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import styled from "styled-components/macro";
+
+function Message() {
```
Comment (lukemt) [inline src/components/Error.js:3]: Be more precise with the naming. Message sounds like a generic message component. How about NoActivitiesMessage

File: src/components/CardList.js [hunk 2]
```diff
@@ -1,7 +1,11 @@
 import ActivityCard from "./Card";
 import styled from "styled-components/macro";
+import Error from "./Error";
 
 function CardList({ activities, onJoin, onDeleteButtonClick }) {
+  if (!activities.length) {
+    return <Error />;
```
Comment (lukemt) [inline src/components/CardList.js:7]: Importing a component with another name than was defined looks quite confusing ;) 
Error is also to extreme. Call it Hint or Message

**Review by lukemt — COMMENTED** [review]
For you video it's fine ;)

Otherwise, consider my comments :)

File: src/components/Error.js (hunk 1 above)
Comment (macfrei) [inline src/components/Error.js:3]: I would name it `ErrorMessage` or `Error` as well 🙂

**Review by macfrei — APPROVED** [review]
Nice job!

File: src/components/Error.js (hunk 1 above)
Comment (martinhaiss) [reply src/components/Error.js:3]: Thx, changed it.

File: src/components/CardList.js (hunk 2 above)
Comment (martinhaiss) [reply src/components/CardList.js:7]: Done!

File: src/components/Error.js (hunk 1 above)
Comment (martinhaiss) [reply src/components/Error.js:3]: Changed it. Thx!

**Pull request merged**